backend/src/routes/courses/CoursesRouter.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const router = express.Router();
+
+var connection = require('../../../database/database')
+
+router.get('/', (req, res) => {
+  connection.connect(err => {
+    const getCourses = `SELECT * FROM course WHERE Is_Active=TRUE`
+    connection.query(getCourses, (err, result) =>{
+      if (err) {
+        res.send({
+          success: false,
+          message: "An error occured, please try again ",
+        });
+      } else {
+        res.send({
+          success: true,
+          message: "",
+          data: result
+        });
+      } 
+    })
+  })
+})
+
+router.get('/:skillID', (req, res) => {
```

Comment:
Should the route be /:skillID ?
I see this might be a problem if we might make an endpoint for get course by course ID in the future.

Reply:
yup makes sense. thanks for pointing it out